setup.py

```diff
@@ -39,6 +39,7 @@ def get_long_description() -> str:
         "appdirs",
         "toml>=0.9.4",
         "typed-ast>=1.3.1",
+        "regex==2019.8.19",
```

Comment:
I think the idea is that end-user, non-dev dependencies should not be pinned and should be as loose as possible. Does it work like this?

```suggestion
        "regex",
```

Reply:
yes it should work, i'll change